hack/examples/bases/nutanix/clusterclass/kustomization.yaml.tmpl

```
@@ -5,7 +5,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://raw.githubusercontent.com/nutanix-cloud-native/cluster-api-provider-nutanix/1a7cd69ba35de01e56dcf2dda7f31973111d2317/templates/cluster-template-clusterclass.yaml
+- https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/releases/download/v1.4.0-alpha.2/cluster-template-clusterclass.yaml
```

can we use `CAPX_VERSION` so that makefile target can take care of updating the version.
```suggestion
- https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/releases/download/${CAPX_VERSION}/cluster-template-clusterclass.yaml
```
https://github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/blob/main/make/examples.mk#L8